src/sagas/admin.js

```diff
@@ -47,6 +75,7 @@ function sendTxWrapper({ txObject, gasPrice, gasLimit, fromAddress }) {
         if (err) {
           reject(err);
         }
+        toast(message);
         resolve(res);
```

Comment:
I think you can check here if the user has rejected the transaction and if so, throw an error with reject.